### PR TITLE
docs: sync basic instructions

### LINKED ISSUE #140: docs: update the development branch docs
- [x] instructions to spin up the platform should be in line with current development branches
- [x] rework the "First Steps" section for new CLI and UI

### PR DESCRIPTION
**Pull request by rokroskar**
Provide some basic instructions for the development branch that are in sync with the recent changes. 

---
addresses #135 
closes #140

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -21,8 +21,8 @@ RENGA (連歌)
 .. image:: https://img.shields.io/travis/SwissDataScienceCenter/renga.svg
    :target: https://travis-ci.org/SwissDataScienceCenter/renga
 
-.. image:: https://readthedocs.org/projects/renga/badge/?version=latest
```
Comment (jirikuncar) [inline README.rst]: I would suggest to keep all on latest and merge development to master ASAP, keeping the old master as `maint-0.1` (cc @erbou)

File: docs/user/firststeps.rst [hunk 2]
```diff
@@ -30,137 +27,130 @@ Our first Renga project
...
-
-This creates a context - we can now run it on an engine:
-
-.. code-block:: console
-
-    $ renga contexts run 14971456 docker
-    16d1491b
-
-To see all of your existing contexts:
+::
```
Comment (jirikuncar) [inline docs/user/firststeps.rst]: `..code-block:: console`

File: docs/user/firststeps.rst [hunk 3]
```diff
@@ -7,16 +7,13 @@ First Steps
...
 
-To try out Renga, you first need a platform to connect to: see
-:ref:`setup` for instructions on how to get one running on your own
-machine in a few minutes.
-
 Interaction with the platform takes place via the Python-based
 command-line interface (CLI) and the Python API. You can get both via pip:
 
-.. code-block:: console
+::
```
Comment (jirikuncar) [inline docs/user/firststeps.rst]: `.. code-block:: console`

File: docs/user/firststeps.rst [hunk 4]
```diff
@@ -30,137 +27,130 @@ Our first Renga project
...
 
-.. code-block:: console
+    $ mkdir -p ~/renga-projects/myproject
+    $ cd ~/renga-projects/myproject
 
-    $ renga contexts list
-    ID               VERTEX_ID    IMAGE
-    ---------------  -----------  -----------
-    16d1491b         4168         hello-world
+Initialize the project as a Renga project:
```
Comment (jirikuncar) [inline docs/user/firststeps.rst]: IMHO here we should point to `renga-python` documentation and not repeat ourself

Comment (rokroskar) [reply docs/user/firststeps.rst]: Right, but I think we do want some sort of "First steps" other than a link in the main repo as well?

File: README.rst (hunk 1 above)
Comment (rokroskar) [reply README.rst]: that's fine with me -- we should have a conclusion on #138 before doing this I suppose

**Comment by erbou** [comment]
PR is fine with me for a merge on development - I tagged master/615132e as 'maint-0.1' but wait before merging dev to master. The master is currently used in production and we need to provide support for it.

File: docs/user/firststeps.rst (hunk 4 above)
Comment (rokroskar) [reply docs/user/firststeps.rst]: also, the "first steps" in the main renga repo should soon contain instructions that include the UI etc. and we want all of this to be on the same page so the user doesn't have to click around too much

**Pull request merged**